cli/app/app.go

```diff
@@ -1165,9 +1182,9 @@ func Convert(c *cli.Context) {
 func checkUnsupportedKey(service interface{}) {
 	s := structs.New(service)
 	for _, f := range s.Fields() {
-		if f.IsExported() && !f.IsZero() {
+		if f.IsExported() && !f.IsZero() && strings.Compare(f.Name(), "Networks") != 0 {
```

Comment:
`!=` makes this much more easier to read.
Also [docs](https://golang.org/pkg/strings/#Compare) says: "Compare is included only for symmetry with package bytes. It is usually clearer and always faster to use the built-in string comparison operators ==, <, >, and so on."